docs/source/ecf.rst

```diff
@@ -406,3 +406,150 @@ The path to the directory containing the Stage 3 JWST data.
+
+run_nwalkers
+''''''''''''
+Integer. The number of walkers to use.
+
+run_nburn
+'''''''''
+Integer. The number of burn-in steps to run.
+
+
+Dynesty Fitting Parameters
+''''''''''''''''''''''''''
+The following set the parameters for running dynesty. These options are described in more detail in: https://dynesty.readthedocs.io/en/latest/api.html?highlight=unif#module-dynesty.dynesty
+
+run_nlive
+'''''''''
+Integer. Number of live points for dynesty to use.
+
+run_bound
+'''''''''
+The bounding method to use. Options are: ['none', 'single', 'multi', 'balls', 'cubes']
+
+run_sample
+''''''''''
+The sampling method to use. Options are ['auto', 'unif', 'rwalk', 'rstagger', 'slice', 'rslice', 'hslice']
+
+run_tol
+'''''''
+Float. The tolerance for the dynesty run. Determines the stopping criterion. The run will stop when the estimated contribution of the remaining prior volume tot he total evidence falls below this threshold.
+
+
+interp
+''''''
+Boolean to determine whether the astrophysical model is interpolated when plotted. This is useful when there is uneven sampling in the observed data.
+
+isplots_S5
+'''''''''''
+Sets how many plots should be saved when running Stage 5.
+
+- ``isplots_S5 >= 1``:
+   - (All Fitters) Plots the fitted model for each spectroscopic lightcurve.
+
+- ``isplots_S5 >= 3``:
+   - (All Fitters) Plot RMS deviation, residuals distribution.
+   - (emcee) Plot chains from the emcee sampler.
+
+- ``isplots_S5 >= 5``:
+   - (emcee, dynesty) Plot posterior corner plots.
+
+
+hide_plots
+'''''''''''
+If True, plots will automatically be closed rather than popping up on the screen.
+
+
+topdir + inputdir
+''''''''''''''''''
+The path to the directory containing the Stage 4 JWST data.
+
+
+topdir + outputdir
+'''''''''''''''''''
+The path to the directory in which to output the Stage 5 JWST data and plots.
+
+
+Stage 5 Fit Parameters
+----------------------
+
+This file describes the transit/eclipse parameters and their prior distributions. Each line describes a new parameter, with the following basic format:
+
+``Name    Value    Free    PriorPar1    PriorPar2    PriorType``
+
+The ``PriorType`` can be U (Uniform), LU (Log Uniform), or N (Normal). If U/LU, then ``PriorPar1`` and ``PriorPar2`` are the lower and upper limits of the prior distribution. If N, then ``PriorPar1`` is the mean and ``PriorPar2`` is the stadard deviation of the Gaussian prior.
+
+``Free`` determines whether the parameter is fixed, free, independent, or shared. Fixed parameters are fixed in the fitting routine and not fit for. Free parameters are fit for according to the specified prior distribution, independently for each wavelength channel. Shared parameters are fit for according to the specified prior distribution, but are common to all wavelength channels. Independent variables set auxiliary functions needed for the fitting routines.
+
+Here's an example fit parameter file:
+
+
+.. include:: ../media/S5_fit_par_template.ecf
```

Comment:
If there is still the transittype line in that fitpar template (and the one in demos) can you please delete it

Reply:
Ah, yeah. It's still there. Deleting now.